docs-site/docs/configuration/pipelines.md

```diff
@@ -0,0 +1,46 @@
+---
+sidebar_position: 2
+id: pipelines
+title: Registering Pipelines
+---
+
+Pipelines must be registered in IRIDA Next to be able to launch workflows. They are registered when the server is started up and Future restarts of the server will only register new pipelines that are added to the configuration file, and schema files for a particular pipeline will update if newer versions are available in the repository.
+
+## Pipeline repository
+
+Currently only **Nextflow** pipelines are supported and they must have a Github repository. Each pipeline is required to have a `nextflow_schema.json` file at the top level of the repository, and a `schema_input.json` file under an `assets` directory within the repository.
+
+## Setup
+
+The configuration file to register pipelines is in `json` format and stored in the `config/pipelines/` directory with the name `pipelines.json`.
+
+This `pipelines.json` file should be in the format below and must include the following:
+
+- **URL** of the pipeline Github repository
+- **name** of the pipeline
+- **description** of the pipeline
+- **versions** of the pipeline that should be available to launch
```

Comment:
**nitpick (blocking):** Could you make note here that `name` in `versions` refer to the `-r` flag in nextflow. From the nextflow docs this is:

```
-r (master)
Revision to clone - It can be a git branch, tag, or revision number.
```

https://www.nextflow.io/docs/latest/cli.html